### DISCUSSION TITLE
NASA發射飛行探測器撞擊小行星！🪐

### DISCUSSION THREAD

**Discussion by bonny123865** [1101-GS4719-C/Python-Community-GS4719-C#712] [General]
原文來源:
https://www.bbc.com/zhongwen/trad/science-63043375

新聞摘要:
美國航天機構的「飛鏢」（Dart）飛行探索器直接撞進一枚小行星，過程中自身完全毀滅。
「飛鏢」的攝像頭每秒傳回一張照片，一直到與目標相撞前的一刻——撞擊的目標是一個寬度為160米、名為「迪莫弗斯」（Dimorphos）的物體。
這項技術稱為』動力撞擊器技術『（kinetic impactor technique），可能被用於防備日後可能有小行星飛來的時候。這是一個非常簡單的想法：你把飛行器飛進你所擔心的軌道上，然後你用飛行器的體積和速度來稍為改變物體的軌道，以足夠讓它躲開地球，而不是撞上地球。

個人心得:
昨天晚上看到這個新聞的時候，從飛鏢的攝影機越接近越能清楚看到那個小行星的表面，看起來就像表面粗糙的大石頭！
若是之後有其他小行星預計會碰撞到地球時，希望這個技術能讓地球受到保護！也希望後續對於另一顆更大的小行星的研究計畫能順利完成！
![image](https://user-images.githubusercontent.com/90883526/192680282-a547bf76-e076-4312-bdf1-bf982f40a550.png)

**Comment by Yunhans** [comment]
會不會之後會有像電動那樣操控探索器
把小行星射爛

**Reply by bonny123865** [reply]
等等 那麼危險先不要啦
我比較希望有國家主動先解決太空垃圾的問題!
射爛太空垃圾感覺比較好?!

**Comment by Chiway1229** [comment]
昨天有看到google上針對這件事情有個小彩蛋，大家都可以去查查喔

**Comment by danwan1027** [comment]
希望這個技術能快點發展起來，讓地球免於宇宙中的潛在危害

**Comment by AnAnistired** [comment]
是太空版薩德系統的概念嗎😍

**Comment by williamyen0623** [comment]
這技術看起來對人類防衛地球似乎是個不錯的選擇，希望今後能發展起來

**Comment by WEI-CHEN-CHUN** [comment]
目前人類從發現有小行星靠近地球到派飛行器的時間，需相當長的時間。若真的發生高速的小行星或系外進入的，尚不足以改變其飛行路徑

**Comment by ABCDEFGHIJKLMNO12345678** [comment]
相信如果這項技術發展完全的話 對人類的安全會有很大的助益

**Comment by Zoe06060606** [comment]
希望此技術能快點發展建全 使地球免於危險之中

**Comment by zhiron544** [comment]
我覺得這項技術是一種對地球的保護，也是一種對未知的挑戰。我希望這項技術能夠有效地避免可能的小行星撞擊
